src/routes/contentful/home.json.ts

```diff
@@ -14,6 +14,10 @@ export const get = async (): Promise<EndpointOutput<HomePage>> => {
 					title
 					description
 				}
+				primaryLink
```

Comment:
I can see that the field `topBannerSubtitle` is disabled in the CMS, but you're still querying it. This causes the build to fail. See the Netlify build logs.